dispatch/dispatch.go

```diff
@@ -341,14 +201,9 @@ func (d Dispatcher) Update(norn model.Norn) {
 }
 
 func (d Dispatcher) Stop(context.Context) error {
-	close(d.DispatchQueue.Load().(chan *eventWithID))
-
 	d.Mutex.Lock()
 	d.Measures.DeliverUntilGauge.Set(float64(d.Norn.ExpiresAt))
```

Comment:
```suggestion
```

The dispatcher shouldn't have a deliver until gauge - that's specific to each norn.  I'm not sure we need a stop function for dispatchers, since even for SQS we don't have to close a connection or anything.